src/core/Do.Architecture/Architecture/ILayer.cs

```diff
@@ -2,6 +2,7 @@
 
 public interface ILayer
 {
+    string Id => GetType().Name;
```

Comment:
leave it as `string Id { get; }`, do not introduce a default implementation here